CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
 * [CHANGE] Memberlist: cluster label verification feature (`-memberlist.cluster-label` and `-memberlist.cluster-label-verification-disabled`) is now marked as stable. #3108
 * [CHANGE] Distributor: only single per-tenant forwarding endpoint can be configured now. Support for per-rule endpoint has been removed. #3095
 * [CHANGE] Query-frontend: truncate queries based on the configured blocks retention period (`-compactor.blocks-retention-period`) to avoid querying past this period. #3134
+* [CHANGE] Query-frontend: truncate queries based on the configured creation grace period (`--validation.create-grace-period`) to avoid querying past too far into the future. #3172
```

Comment:
```suggestion
* [CHANGE] Query-frontend: truncate queries based on the configured creation grace period (`--validation.create-grace-period`) to avoid querying too far into the future. #3172
```